src/V8JsiRuntime_impl.h

```diff
@@ -158,10 +158,14 @@ class V8Runtime : public facebook::jsi::Runtime {
 #endif
 
  public: // Used by NAPI implementation
-  v8::Global<v8::Context> &GetContext() {
+  inline v8::Global<v8::Context> &GetContext() {
```

Comment:
>inline [](http://example.com/codeflow?start=2&length=6)

(nit) Why declaring explicit "inline"? Won't compiler inline as needed anyways?

Reply:
I think you're right, it has no effect since it's a member function. But I wanted to be explicit, especially since this is a header.